content/en/docs/examples/bookinfo/index.md

```diff
@@ -67,29 +71,88 @@ and outgoing calls for the services, providing the hooks needed to externally co
 via the Istio control plane, routing, telemetry collection, and policy enforcement
 for the application as a whole.
 
+{{< /tab >}}
+
+{{< tab name="Ambient Mode" category-value="ambient-mode" >}}
+
+In Ambient mode, ztunnel proxies are deployed on each node.
+We can also put Waypoint proxies in front of each service
+The resulting deployment will look like this:
+
+FIXMEFIXMEFIXME ADD IMAGE
+
+All of the microservices incoming and outgoing traffic will be intercepted by the ztunnel proxies,
+providing the hooks needed to externally control, L4 routing, telemetry collection, and policy enforcement
+
+Further, the ztunnel proxy will forward the traffic to a Waypoint proxy,
+which provide the hooks needed to externally control, L7 routing, telemetry collection, and policy enforcement.
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
 ### Start the application services
 
 {{< tip >}}
-If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
+If you using a cloud provider, please ensure your cluster has at least 4 nodes, each with 4GB of memory and 4 CPUS.
```

Comment:
```suggestion
If you using a cloud provider, please ensure your cluster has at least 4 nodes, each with at least 4GB of memory and 4 CPUS.
```

Reply:
"CPUs".

TBH this is really high for an app that does ~nothing.  Is this still a relevant recommendation, and is it to do with reservations?

Reply:
Honestly, I just threw a number out there that I thought would be around that of a "Standard GKE Node". Open to suggestions